students/shervs/lesson04/trigrams.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import random
+import sys
+
+def read_in_data(in_filename):
+    with open(in_filename) as infile:
+        in_lines = infile.readlines()
+    return in_lines
+
+def make_words(in_lines):
+    in_words = []
+    for line in in_lines:
+        #Remove headers and footers
+        if line[0:3] == '***':
+            in_lines.remove(line)
+        #Remove punctuation
+        else:
+            new_line = line.translate(str.maketrans(
+                    '''0123456789!"#$%&()*+,-./:;<=>?@[\]^_`{|}~'''
+                    ,'                                         ' ))
+            #Break each line to wrods and feed into the word list
+            for word in new_line.split():
+                if word is not "I":
```

Comment:
`is` is used with objects comparison like True/False/None and other objects.

for primitive type comparison use `==`